services/webfinger/pkg/config/config.go

```diff
@@ -21,7 +21,7 @@ type Config struct {
 	Instances []Instance `yaml:"instances"`
 	Relations []string   `yaml:"relations" env:"WEBFINGER_RELATIONS" desc:"A comma-separated list of relation URIs or registered relation types to add to webfinger responses."`
 	IDP       string     `yaml:"idp" env:"OCIS_URL;OCIS_OIDC_ISSUER;WEBFINGER_OIDC_ISSUER" desc:"The identity provider href for the openid-discovery relation."`
-	OcisURL   string     `yaml:"ocis_url" env:"OCIS_URL;WEBFINGER_OCIS_URL" desc:"URL, where oCIS is reachable for users."`
+	OcisURL   string     `yaml:"ocis_url" env:"OCIS_URL;WEBFINGER_OWNCLOUD_SERVER_INSTANCE_URL" desc:"The URL for the owncloud server instance relation."`
```

Comment:
huuu, this is dangerous...
the term `owncloud server` is "locked" to oc10 - except you want to exactly say that...

Reply:
yep, that is what the relation is called

Reply:
fixed, set to resolved